examples/visualisation/visualise_data.ipynb

```diff
@@ -65,7 +65,7 @@
    "outputs": [],
    "source": [
     "# set env variable for data\n",
-    "os.environ[\"L5KIT_DATA_FOLDER\"] = \"PATH_TO_YOUR_DATA\"\n",
+    "os.environ[\"L5KIT_DATA_FOLDER\"] = \"/tmp/l5kit_data\"\n",
```

Comment:
Is this change on purpose?

Reply:
Obviously no xD It's the 100th time I fix this